docs/guides/getting-started-with-opentelemetry.md

```diff
@@ -0,0 +1,377 @@
+---
+weight: 5
+title: How to use opentelemetry metrics with VictoriaMetrics
+menu:
+  docs:
+    parent: "guides"
+    weight: 5
+---
+
+# Application monitoring with opentelemetry
```

Comment:
can we add the image something like 

[otel example.excalidraw.zip](https://github.com/VictoriaMetrics/VictoriaMetrics/files/14397988/otel.example.excalidraw.zip)